packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/utils.js

```diff
@@ -394,3 +394,36 @@ export function build_expression(context, expression, metadata, state = context.
 
 	return sequence;
 }
+
+/**
+ * Wraps a statement/expression with dev stack tracking in dev mode
+ * @param {CallExpression} call_expression - The function call to wrap (e.g., $.if, $.each, etc.)
+ * @param {{ start?: number }} node - AST node for location info
+ * @param {'component' | 'if' | 'each' | 'await' | 'key'} type - Type of block/component
+ * @param {Record<string, number | string>} [additional] - Any additional properties to add to the dev stack entry
+ * @returns {Statement} - Statement with or without dev stack wrapping
+ */
+export function with_dev_stack(call_expression, node, type, additional) {
+	if (!dev) {
+		return b.stmt(call_expression);
+	}
+
+	const location = node.start && locator(node.start);
+	if (!location) {
+		return b.stmt(call_expression);
+	}
+
+	return b.stmt(
+		b.call(
+			'$.with_dev_stack',
+			b.arrow([], b.block([b.stmt(call_expression)])),
```

Comment:
using an implicit return will keep the statement on a single line in many cases 

```suggestion
			b.arrow([], call_expression),
```